### PR TITLE
Fix selections for project overview

### LINKED ISSUE #37: When viewing projects from other users also archived projects are shown

### PR DESCRIPTION
**Pull request by s3inlc**
Archived/own projects are correctly shown when using checkboxes

closes #37

### CODE REVIEW

File: views/project/detail.php [hunk 1]
```diff
@@ -189,7 +189,7 @@ function submitData() {
                     </div>
                 </div>
 
-                <?php if($data['project']->getUserId() == $data['loginUser']){ ?>
+                <?php if($data['project']->getUserId() == $data['loginUser'] || $auth->isSuperAdmin()){ ?>
```
Comment (vogti) [inline views/project/detail.php]: Why SuperAdmin? Should better be Admin.

File: controllers/project.php [hunk 2]
```diff
@@ -189,7 +185,7 @@ public function detail() {
                     }
                 }
             } // archive project
-            else if (!empty($this->get['archive']) && $this->get['archive'] == true) {
+            else if (!empty($this->get['archive']) && $this->get['archive'] == true && ($project->getUserId() == $auth->getUserID() || $auth->isSuperAdmin())) {
```
Comment (vogti) [inline controllers/project.php]: Why SuperAdmin? Should better be Admin.

File: controllers/project.php [hunk 3]
```diff
@@ -169,15 +165,15 @@ public function detail() {
             }
 
             // remove member
-            if (isset($this->get['remove']) && $project->getUserId() == $auth->getUserID()) {
+            if (isset($this->get['remove']) && ($project->getUserId() == $auth->getUserID() || $auth->isSuperAdmin())) {
```
Comment (vogti) [inline controllers/project.php]: Why SuperAdmin? Should better be Admin.

File: controllers/project.php [hunk 4]
```diff
@@ -169,15 +165,15 @@ public function detail() {
...
                 $user = $FACTORIES::getUserFactory()->get($this->get['remove']);
                 if ($user != null) {
                     $qF1 = new QueryFilter(ProjectUser::USER_ID, $user->getId(), "=");
                     $qF2 = new QueryFilter(ProjectUser::PROJECT_ID, $project->getId(), "=");
                     $FACTORIES::getProjectUserFactory()->massDeletion(array($FACTORIES::FILTER => array($qF1, $qF2)));
                 }
             } // add member
-            else if (isset($this->post['member']) && $project->getUserId() == $auth->getUserID()) {
+            else if (isset($this->post['member']) && ($project->getUserId() == $auth->getUserID() || $auth->isSuperAdmin())) {
```
Comment (vogti) [inline controllers/project.php]: Why SuperAdmin? Should better be Admin.

**Comment by s3inlc** [comment]
requested changes are made

**Pull request merged**